### PR TITLE
Swipe feature url param

### PR DESCRIPTION
**Pull request by DenverCoder544**
-Add layerSwipe = true to created link when said tool is activated.
-Turn on layerSwipe by url param.

image

### CODE REVIEW

**Comment by ZakarFin** [comment]
There are some changes to layerswipe state handling (`state.active`) and activating it on startup on #2543. Lets wait merging this until that one is completed.

**Comment by ZakarFin** [comment]
I think we should make layerswipe stateful and implement the getStateParams() function for it instead of adding the param to link generating function. But there are a bunch of changes for layerswipe and state handling already on https://github.com/oskariorg/oskari-frontend/tree/embedded-layerswipe so it's best to wait until that one is compeleted before merging or changing these.

File: bundles/mapping/layerswipe/instance.js [hunk 1]
```diff
@@ -55,6 +55,9 @@ Oskari.clazz.define(
                 }
             };
             sandbox.request(this, addToolButtonBuilder('LayerSwipe', 'basictools', buttonConf));
+            if (this.state?.active) {
+                this.setActive(true);
+            }
```
Comment (ZakarFin) [inline bundles/mapping/layerswipe/instance.js:60]: This is probably no longer needed after #2550?

File: bundles/mapping/toolbar/default-buttons.js [hunk 2]
```diff
@@ -229,13 +229,14 @@ Oskari.clazz.category(
             const linkParams = sandbox.generateMapLinkParameters({});
             const viewUuid = this._getLinkUuid();
 
+            const layerSwipeActive = sandbox?.findRegisteredModuleInstance('LayerSwipe')?.active;
```
Comment (ZakarFin) [inline bundles/mapping/toolbar/default-buttons.js:232]: And this is outdated as well after #2550. We should implement the getStateParameters() function for layerswipe instead as it is now stateful. This way we don't need to handle it separately on the link creation.

**Comment by ZakarFin** [comment]
The parameter is now "swipe" on oskari-server: https://github.com/oskariorg/oskari-server/blob/develop/control-base/src/main/java/fi/nls/oskari/control/view/modifier/param/LayerSwipeParamHandler.java#L13

**Comment by DenverCoder544** [comment]
Made a new pr on top of recent other changes on the matter, this pr is now obsolete.

https://github.com/oskariorg/oskari-frontend/pull/2551

**Pull request closed without merging**